src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+public class Todo extends Task{
+
+    protected final String SYMBOL = new String("T");
+
+    public Todo(String description, boolean isDone) {
+        super(description, isDone);
+    }
+
+    public String getSYMBOL() {
+        return SYMBOL;
+    }
+
+    @Override
+    public String print() {
+        return ("[" + this.getSYMBOL() + "]" + super.print());
+    }
+
+    @Override
+    public String printToFile() {
+        String isDoneStr = new String((this.isDone) ? "1" : "0");
+        String s = new String(SYMBOL + "|" + isDoneStr + "|" + this.description + "\n");
+        return s;
```

Comment:
Similarly, you could also consider assigning a more descriptive name to the string here.